### PR TITLE
Rapidpro org settings

### PR DESCRIPTION
**Pull request by DevChima**
Rapidpro=None

### CODE REVIEW

File: ndoh_hub/settings.py [hunk 1]
```diff
@@ -350,7 +350,12 @@
...
 
 # ADA
+ADA_SYMPTOMCHECK_V2 = env.bool("ADA_SYMPTOMCHECK_V2", False)
 ADA_RAPIDPRO_URL = env.str("ADA_RAPIDPRO_URL", None)
 ADA_RAPIDPRO_TOKEN = env.str("ADA_RAPIDPRO_TOKEN", None)
 
 ADA_PROTOTYPE_SURVEY_FLOW_ID = env.str("ADA_PROTOTYPE_SURVEY_FLOW_ID", None)
+
+if ADA_SYMPTOMCHECK_V2:
+    RAPIDPRO_URL = env.str("RAPIDPRO_URL")
```
Comment (erikh360) [inline ndoh_hub/settings.py]: are these different to the rapidpro settings on line 354?

and is it necessary to have the `ADA_SYMPTOMCHECK_V2` flag?

**Review by erikh360 — COMMENTED** [review]
just the two questions.

also I'm a bit hesitant to merge this without fixing the tests, we're bound to miss something if the tests are failing

File: ada/utils.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from __future__ import absolute_import, division
+
+from django.conf import settings
+from temba_client.v2 import TembaClient
+
+rapidpro = None
+if settings.EXTERNAL_REGISTRATIONS_V2:
```
Comment (erikh360) [inline ada/utils.py]: I don't think we need this, maybe check if RAPIDPRO_URL and RAPIDPRO_TOKEN is available

**Review by erikh360 — APPROVED** [review]
just the one

File: ada/utils.py (hunk 2 above)
Comment (DevChima) [reply ada/utils.py]: Fixed

**Review by erikh360 — APPROVED** [review]

**Pull request merged**